redis-om-spring/src/main/java/com/redis/om/spring/annotations/Vectorize.java

```diff
@@ -0,0 +1,11 @@
+package com.redis.om.spring.annotations;
+
+import java.lang.annotation.*;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.FIELD, ElementType.ANNOTATION_TYPE })
+public @interface Vectorize {
+  String destination() default "";
```

Comment:
I think that destination property must be mandatory as we always need to store the vectorized data in a byte array.